plugins/palo_alto_pan_os/requirements.txt

```diff
@@ -4,8 +4,8 @@
 gunicorn==22.0.0
 xmltodict==0.12.0
 dicttoxml==1.7.4
-validators==0.22.0
+validators==0.34.0
 IPy==1.01
-parameterized==0.8.1
-jsonschema==3.2.0
+parameterized==0.9.0
+jsonschema==4.21.1
```

Comment:
Isn't that included to the SDK. If so, I think we can remove it from here.

Reply:
I am not too sure. Is it? I know `requests` is but not too sure about `jsonschema`?